deployment/hasura/migrations/AerieScheduler/1_optional_sim_per_goal/up.sql

```diff
@@ -0,0 +1,6 @@
+alter table scheduling_specification_goals add column simulate_after boolean not null default true;
+
+comment on column scheduling_specification_goals.simulate_after is e''
+  'Whether to re-simulate after evaluating this goal and before the next goal.';
+
+call migrations.mark_migration_applied('1');
```

Comment:
This call also needs to happen at the bottom of `scheduler-server/sql/scheduler/applied_migrations.sql`.

Reply:
done